master/master.cfg

```diff
@@ -872,7 +872,7 @@ def add_halide_cmake_test_steps(factory, builder_type):
 
     source_dir = get_halide_source_path()
     build_dir = get_halide_build_path()
-    install_dir = get_halide_install_path()
+    install_dir = get_halide_install_path()  # NOQA
```

Comment:
What does this comment do here (presumably ignores some warning)?

Reply:
That variable is unused.